programming/sed-article.md

```diff
@@ -265,7 +253,7 @@ final underscore in the pattern wouldn't have anything to match.
 Another approach we could try is `/_[^_]+_/`.
 This looks for an underscore, followed by any characters
 that are not underscores, followed by an underscore.
-It matches "\_apple\_", but not "\_apple_banana_cherry\_".
+It matches "\_apple\_", but not "\_apple\_banana\_cherry\_".
```

Comment:
Is this changes just for markdown syntax so it doesn't italicize the text?

Reply:
Same.